lib/idx/remediate.ts

```diff
@@ -67,15 +67,15 @@ export async function remediate(
   values: RemediationValues,
   options: RemediateOptions
 ): Promise<RemediationResponse> {
-  let { neededToProceed, interactionCode } = idxResponse;
+  let { neededToProceed, interactionCode, context } = idxResponse;
   const { flow } = options;
 
   // If the response contains an interaction code, there is no need to remediate
   if (interactionCode) {
     return { idxResponse };
   }
 
-  const remediator = getRemediator(neededToProceed, values, options);
+  const remediator = getRemediator(neededToProceed, values, options, context);
```

Comment:
Might be a little cleaner/easier to have `getRemediator` accept the entire `idxResponse` object, rather than `neededToProceed` and `context` separately

```js
const remediator = getRemediator(idxResponse, values, options);
```

Reply:
Done